notebooks/example.ipynb

```diff
@@ -200,6 +199,143 @@
     "%%bash\n",
     "cat ./fixtures/rendered_config.yaml"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea5eff92",
+   "metadata": {},
+   "source": [
+    "## Example 3: Comparing two config files\n",
+    "Let's explore using the at using the `config.compare()` method to compare two two config files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71d27692",
+   "metadata": {},
+   "source": [
+    "We again need to start by importing the `uwtools.api.config` from the uwtools python package."
```

Comment:
This may or may not be good for instructional purposes, but IIRC this shouldn't be necessary in a Jupyter notebook given that we've imported this already in a previous cell. Some instructional notebooks I've worked with have a cell near the top of the notebook that does a bunch of imports that will be needed to support cells that follow; I'm not necessarily asking for that, but it's something we could consider if we found that executing (or repeating) `import` statements as we go distracted from the story. Thoughts?

Reply:
My thought was if someone was jumping around the notebook looking for the specific example for the tool they were using, it shows a true start to finish? I see your point of it cluttering up the notebook, I am fine with removing it.

Reply:
I don't feel strongly about it, and you make a good point. Your call.